### DISCUSSION TITLE
eÁfa tesztelés

### DISCUSSION THREAD

**Discussion by RuszinZsolt** [nav-gov-hu/eVAT#65] [Kérdések és válaszok - Gépi interfész]
Elindult a https://eafa-test.nav.gov.hu, ami klassz, de egyelőre egy az online számla tesztfelületére vezet vissza, holott már van, aki megszüntette az egyéni vállalkozását és a január számla bevallása kapcsán megnéztük volna a tesztfelületet. 

Tudomásom szerint még egyetlen könyvelőszoftver sincs kész a saját eÁfa felületével és számos kérdést feltettek, de nem kaptak válaszokat. Kértem én is a https://nav.gov.hu/ado/eafa/teszteles/tesztelheto-az-eafa instrukciói szerint gép-gép kapcsolathoz hozzáférést és kaptam is, de ez úgy derült ki, hogy rákérdeztem és akkor visszahívtak és mondták, hogy már tegnap beállították. Jobb lett volna egy válasz email.

Jó lenne, ha az eÁfa összeállítani a kimenő számlák alapján az A60-asokat is (sok vállalkozásnak csak kimenő EU számlája van).

**Comment by RuszinZsolt** [comment]
Most, hogy jeleztem, kiküldték mindenkinek e-mailben, hogy mehet a tesztelés. Köszönjük.

**Comment by renced42** [comment]
Kedves @RuszinZsolt 

Köszönjük a javaslatot.

Az eÁfa rendszerbe történő bejelentkezéskor az Online Számla oldalára navigál a rendszer, mivel ott lehet bejelentkezni KAÜ vagy másodlagos felhasználóval. A bejelentkezést követően pedig automatikusan vissza irányít az eÁfa rendszer felületére, vagy arra a felületre ahol a belépést kezdeményezték.

A rendszerek között a főoldalon található Alkalmazások menü pontban található ikonokkal lehet váltani.
A teszt környezetben az adózói adatokat nem frissítjük folyamatosan, de szükség esetén kérésre megtesszük, ha valahol arra szükség van.

A könyvelő szoftverek tekintetében még javában zajlanak a tesztelések, de valószínű, hogy sokan nem foglalkoztak még a témával. Azokat a kérdéseket amik beérkeznek igyekszünk megválaszolni. Bár vannak meg nem válaszolt kérdések is, de ezeket is hamarosan kipörgetjük :)

A teszt rendszerben az adózói adatokat nem frissítjük naprakészen, így lehet, hogy a valóságtól eltérnek egyes adózói adatok vagy státuszok.
Természetesen szükség esetén ezeket adózói kérésre tudjuk frissíteni.

**Comment by RuszinZsolt** [comment]
Milyen szép lenne, ha így lenne. A https://eafa-test.nav.gov.hu/ oldalról elindulva és a https://onlineszamla-test.nav.gov.hu/login?source=EAR oldalon KAÜ azonosítást követően nem történik visszairányítás. Ehhez ki kell választani azt az adózót, akire a tesztelés vonatkozik. Ekkor valóban megnyílik végre a https://eafa-test.nav.gov.hu/ 

Végre elkezdhetem a tesztelést és az összevetést. Most már csak az kell, hogy a NAV OSA ne álljon le minden másnap. Jó lenne, ha **január 8-án** lenne adatfrissítés, hogy láthassam, hogy a tesztadatok és a valóság között egy pofonegyszerű esetben megvan-e az egyezőség. Köszönöm.

**Comment by renced42** [comment]
> Milyen szép lenne, ha így lenne. A https://eafa-test.nav.gov.hu/ oldalról elindulva és a https://onlineszamla-test.nav.gov.hu/login?source=EAR oldalon KAÜ azonosítást követően nem történik visszairányítás. Ehhez ki kell választani azt az adózót, akire a tesztelés vonatkozik. Ekkor valóban megnyílik végre a https://eafa-test.nav.gov.hu/

Igen ez valóban így van akkor ha a bejelentkezett felhasználóhoz több adózó tartozik és ez így teljesen jó.
